stl/inc/xutility

```diff
@@ -5997,8 +6001,12 @@ _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_inf(const _Ty _Xx) { // constexpr isinf(
 // FUNCTION TEMPLATE _Is_finite
 template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
 _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_finite(const _Ty _Xx) { // constexpr isfinite()
+#if defined(__LDBL_DIG__) && __LDBL_DIG__ == 18
+    return _CSTD _LDtest(&_Xx) == _INFCODE;
+#else // ^^^ 80-bit long double (not supported by MSVC in general, see GH-1316) / 64-bit long double vvv^M

```

Comment:
```suggestion
#else // ^^^ 80-bit long double (not supported by MSVC in general, see GH-1316) / 64-bit long double vvv
```

Reply:
Thank you @sam20908 for catching this. Fixed now.